### PR TITLE
Fix rich text strikethrough

### PR DESCRIPTION
**Pull request by justinvdm**
@Mitso @nathanbegbie ready for review

### CODE REVIEW

**Comment by justinvdm** [comment]
screen shot 2017-01-06 at 5 49 32 pm

File: src/components/RichText/__tests__/RichText-test.js [hunk 1]
```diff
@@ -10,34 +10,34 @@ describe('RichText', () => {
...
+text
 
-        [link](http://www.google.com)
+[link](http://www.google.com)
 
-        *italics*
-        **bold**
-        ***bold italics***
-        ~~strikeout~~
+*italics*
```
Comment (nathanbegbie) [inline src/components/RichText/__tests__/RichText-test.js]: @justinvdm Will italics and bold always be denoted by an asterisk (*) and not underscore (_) ? Is there a particular version/style of markdown that we're using?

Comment (nathanbegbie) [inline src/components/RichText/__tests__/RichText-test.js]: Ah, I see underscores are used for underlining in #321

Comment (justinvdm) [inline src/components/RichText/__tests__/RichText-test.js]: We’re relying on https://github.com/Khan/simple-markdown which supports both, added test for `_` case.

Comment (justinvdm) [inline src/components/RichText/__tests__/RichText-test.js]: screen shot 2017-01-11 at 12 17 14 pm

**Comment by nathanbegbie** [comment]
+1

**Pull request merged**